internal/config/config.go

```diff
@@ -149,6 +149,12 @@ func (c *Config) configArchive() {
 			s3.Port = 443
 		}
 
+		if viper.IsSet("archive.region") {
+			s3.Region = viper.GetString("archive.region")
+		} else {
+			s3.Region = "us-east-1"
+		}
+
 		if viper.IsSet("archive.chunksize") {
 			s3.Chunksize = viper.GetInt("archive.chunksize") * 1024 * 1024
 		}
```

Comment:
Can we have the default for `chunksize`/`bufsize` here rather than cases for it it the ingest?